samseg/setup.py

```diff
@@ -15,18 +16,26 @@
 VERSION = '0.1.1'
 
 REQUIRED = [
-    'nibabel >= 2.2.1'
+    'nibabel >= 2.2.1',
     'numpy >= 1.13.3',
     'scipy >= 1.0.0',
     'gems2python',
 ]
 
+PACKAGES_TO_FIND = ['samseg', 'samseg.dev_utils']
+NO_VISUALIZATION_ARG = '--no-visualization'
+if not NO_VISUALIZATION_ARG in sys.argv:
```

Comment:
note that you can also do `if NO_VISUALIZATION_ARG not in sys.argv:`

Reply:
Yes. Reads more naturally that way.